docker/sawtooth-pdo.proxy.yaml

```diff
@@ -35,7 +35,7 @@ services:
     environment:
       http_proxy: "${http_proxy}"
       https_proxy: "${https_proxy}"
-      no_proxy: "${no_proxy},shell,rest-api,validator,settings-tp,pdo-tp,pdo-build"
+      no_proxy: "${no_proxy},shell,rest-api,validator,settings-tp,pdo-tp,pdo-build,sawtooth-pdo-build,pdo-sgx-build"
 
```

Comment:
I guess the same comment above applies here as well. Should not need pdo-sgx-build as part of no-proxy.

Reply:
same response. there should be ONE value across all the containers so we don't have to keep maintaining multiple lists.